quiz-bundle-spec.md

```diff
@@ -0,0 +1,71 @@
+# Qwiklabs Quiz Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Quiz
+schema_version: 1
+
+# The authoring title of the quiz - not visible to learners.
+title: Super Hard Quiz
+
+# The threshold grade that a student needs to achieve in order to count as "passing" the quiz.
+passing_percentage: 67
+
+# The "items" (i.e. questions) contained in this quiz.
+items: ...
+```
+
+Note that all of the localized content (stem, option titles, rationales, etc) are HTML content that may be displayed in various contexts. All of these chunks will be sanitized according to the restricted set in the [HTML spec](./html-spec.md).
+
+### Items
+Items are polymorphic - i.e. there are several different _item types_ that are defined slightly differently. `items` is an array of items defined correctly for their type as defined by the `type` attribute.
+
+#### Multiple Choice Items
+
+A quiz item that has multiple options and one answer. One option is the _answer_, and the rest are _distractors_. Each option has a required rationale that explains why that option is correct or incorrect.
+
+attribute               | required | type              | notes
+----------------------- | -------- | ----------------- | -----------------------------------------
+id                      | ✓        | string            | A unique identifier for this item
+type                    | ✓        | 'multiple-choice' | The item type, which is always 'multiple-choice'
+stem                    | ✓        | dictionary        | A locale dictionary of locales for the text that asks the question, such as "Which of the following is a color?"
```

Comment:
You've written "A locale dictionary of locales for..." a lot of places. I think we can assume that that's what a locale dictionary is, and just write something like
```suggestion
stem                    | ✓        | dictionary        | A locale dictionary of the text that asks the question, such as "Which of the following is a color?"
```